Bindings/Java/Matlab/Hopper_Device/EvaluateHopper.m

```diff
@@ -24,7 +24,7 @@
 % Perform a hop with the provided model and evaluate the performance of the
 % hop.
 
-function [score, peakHeight, finalHeight] = EvaluateHopper(hopper, print)
+function [score, peakHeight, finalHeight, heightStruct] = EvaluateHopper(hopper, visualize, print)
```

Comment:
I think we should eventually simplify this so that it returns only the necessary quantities (`score` seems unnecessary). Fine for now, though.